src/SDKs/TrafficManager/TrafficManager.Tests/ScenarioTests/ProfileScenarioTests.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.Management.TrafficManager.Testing.ScenarioTests
     using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
     using Xunit;
 
-    public class ProfileScenarioTests : TestBase
+    public partial class ProfileScenarioTests : TestBase
```

Comment:
@vladca this.CreateResourceGroup does not use the latest ResourceManager.
Even though you have added reference to the new nuget, it's not getting used.
You will have to either use it in this file or change the way ResourceManagement client is created in ResourceGroupHelper.cs